modules/tournament/src/main/crud/CrudForm.scala

```diff
@@ -22,7 +22,7 @@ object CrudForm {
     "clockTime" -> numberInDouble(clockTimePrivateChoices),
     "clockIncrement" -> numberIn(clockIncrementPrivateChoices),
     "minutes" -> number(min = 20, max = 1440),
-    "variant" -> nonEmptyText.verifying(v => guessVariant(v).isDefined),
+    "variant" -> number.verifying(v => variantFromId(v).isDefined),
```

Comment:
This breaks the mobile app and the API https://lichess.org/api#operation/apiTournamentPost

That's why the form accepts both IDs and keys.

Reply:
Ahh, makes sense, didn't know that. Will fix tomorrow!

Reply:
@ornicar Hang on... I thought this is only the CRUD form for moderators. Isn't the code that the API and the mobile app depend on in DataForm.scala? Or am I missing anything else?

Reply:
Oh you're absolutely right. I didn't pay attention to the file name and assumed it was about the user-facing tournament form.